test/src/test/java/hudson/model/QueueTest.java

```diff
@@ -547,8 +547,7 @@ public TestFlyweightTask(AtomicInteger cnt, Label assignedLabel) {
         @Override public int hashCode() {
             return cnt.hashCode();
         }
-        @Override public boolean isBuildBlocked() {return isBlocked;}
```

Comment:
This was only a test task. A real one would already have needed to implement `getCauseOfBlockage` to provide a message to the user. (It was `abstract` prior to https://github.com/jenkinsci/jenkins/pull/2879 in 2.62 so it is unlikely that new code was written against 2.62–2.85 that assumed that only implementing `isBuildBlocked` was OK.)